docker-for-mac/release-notes.md

```diff
@@ -13,10 +13,25 @@ This page contains information about the new features, improvements, known issue
 For information about Edge releases, see the [Edge release notes](edge-release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.3.0.5
+2020-09-15
+
+> [Download](https://hub.docker.com/editions/community/docker-ce-desktop-mac/)
```

Comment:
@StefanScherer @ebriney Should this be desktop.docker.com URL instead?

Reply:
@usha-mandya No that's fine to link to the download page on Docker Hub to get the latest Stable.

Reply:
I used same model as previous PR, but for sure we can think about putting the final url once it's published.
The idea here is to use the hub workflow I suppose, which is good because we can add tutorial links, create an account...